util/liquids.lua

```diff
@@ -19,7 +19,7 @@ local water_texture = "default_water.png"
 local river_water_texture = "default_river_water.png"
 
 local mcla = minetest.get_game_info().id == "mineclonia"
-local mcl2 = minetest.get_game_info().id == "mineclone2"
+local mcl2 = minetest.get_game_info().id == "VoxeLibre"
```

Comment:
Good catch, but this would be better if it did an `or` check if the id was either `mineclone2` or `VoxelLibre` - so that people still playing without having updated to VoxelLibre too don't get their textures broken.